tests/letstest/multitester.py

```diff
@@ -488,9 +487,13 @@ class FabricException(Exception):
                                    security_group_id=security_group_id,
                                    subnet_id=subnet_id)
 
+instances = []
```

Comment:
Since `instance` becomes a global state, can we name it using uppercase letters, and define it at the top of the script?

Reply:
Cancel my comment above, #7057 will move all this logic in a `main` method.